SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetDriver.h

```diff
@@ -175,13 +174,16 @@ class SPATIALGDK_API USpatialNetDriver : public UIpNetDriver
 	TUniquePtr<FSpatialOutputDevice> SpatialOutputDevice;
 
 	TMap<Worker_EntityId_Key, USpatialActorChannel*> EntityToActorChannel;
+	TArray<Worker_OpList*> QueuedStartupOpLists;
```

Comment:
Could it be a local variable in HandleStartupOpQueueing function?

Reply:
no, the queueing runs over multiple frames